docs/en/stack/ml/nlp/ml-nlp-deploy-models.asciidoc

```diff
@@ -23,8 +23,29 @@ Per the <<ml-nlp-overview>>, there are multiple ways that you can use NLP
 features within the {stack}. After you determine which type of NLP task you want
 to perform, you must choose an appropriate trained model.
 
-IMPORTANT: The {stack-ml-features} support transformer models that conform to
-the standard BERT model interface and use the WordPiece tokenization algorithm.
+[IMPORTANT]
+.Supported model architectures
+====
+[[ml-nlp-architectures]]
+
+The {stack-ml-features} support transformer models that conform to the standard
+BERT model interface and use the WordPiece tokenization algorithm.
+
+The current list of supported architectures is:
+
+* BERT
+* DPR bi-encoders with a Torch wrapper
+* DistilBERT with a Torch wrapper
+* ELECTRA
+* MobileBERT
+* RetriBERT
+* MPNet
+* SentenceTransformers bi-encoders with the above transformer architectures and
+a Torch wrapper
+
+Any trained model that uses one of these supported architectures is deployable
```

Comment:
I think a clarification about "with a Torch wrapper" means is in order. Basically this means that eland will wrap an existing model automatically in order to make these architectures compatible with the BERT model interface.

Reply:
I've removed the "with a Torch wrapper" clauses in 749acfcb58f6cc73efdfa752899f218251bcb023